internal/model/game.go

```diff
@@ -1,24 +1,28 @@
 package model
 
+type Game struct {
+	score gameScore
+}
+
 type gameScore struct {
 	BlueScore  int `json:"blueScore"`
 	WhiteScore int `json:"whiteScore"`
 }
 
-func New() *gameScore {
-	return &gameScore{}
+func New() *Game {
+	return &Game{}
 }
 
 func (gameScore *gameScore) ResetScore() {
 	gameScore.BlueScore = 0
 	gameScore.WhiteScore = 0
 }
 
-func (gameScore *gameScore) AddGoal(teamID int) {
+func (game *Game) AddGoal(teamID int) {
```

Comment:
A good practice is keeping the possible short names for the struct variables to which the function is attached. I would propose `g` instead of the `game`.

Example from [the kyma](https://github.com/kyma-project/kyma/blob/main/components/function-controller/internal/git/go2git.go#L55)
Example from [outside of the kyma](https://github.com/kubernetes/kubectl/blob/master/pkg/cmd/config/view.go#L124)